contracts/PausableHelper.sol

```diff
@@ -2,7 +2,7 @@
 
 pragma solidity ^0.8.4;
 
-import "@openzeppelin/contracts/security/Pausable.sol";
```

Comment:
https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4551
![image](https://github.com/The-Poolz/Poolz-Helper/assets/68740472/800d75ff-f5c1-4eef-8955-46a3161045fa)